modules/apps/document-library/document-library-web/src/main/resources/META-INF/resources/image_gallery_display/view_images.jsp

```diff
@@ -91,15 +91,17 @@ DLPortletInstanceSettingsHelper dlPortletInstanceSettingsHelper = new DLPortletI
 					thumbnailSrc = DLURLHelperUtil.getThumbnailSrc(fileEntry, fileVersion, themeDisplay);
 				}
 
+				String description = fileEntry.getDescription();
+
 				String title = fileEntry.getTitle();
 
-				if (Validator.isNotNull(fileEntry.getDescription())) {
-					title += " - " + fileEntry.getDescription();
+				if (Validator.isNotNull(description)) {
+					title += " - " + description;
 				}
 				%>
 
 				<liferay-ui:search-container-column-text>
-					<div class="image-link preview" <%= (hasAudio || hasVideo) ? "data-options=\"height=" + playerHeight + "&thumbnailURL=" + HtmlUtil.escapeURL(DLURLHelperUtil.getPreviewURL(fileEntry, fileVersion, themeDisplay, "&videoThumbnail=1")) + "&width=640" + dataOptions + "\"" : StringPool.BLANK %> href="<%= imageURL %>" tabindex="0" thumbnailId="<%= thumbnailId %>" title="<%= title %>">
+					<div class="image-link preview" <%= (hasAudio || hasVideo) ? "data-options=\"height=" + playerHeight + "&thumbnailURL=" + HtmlUtil.escapeURL(DLURLHelperUtil.getPreviewURL(fileEntry, fileVersion, themeDisplay, "&videoThumbnail=1")) + "&width=640" + dataOptions + "\"" : StringPool.BLANK %> description="<%= description %>" href="<%= imageURL %>" tabindex="0" thumbnailId="<%= thumbnailId %>" title="<%= title %>">
```

Comment:
Hey @antonio-ortega, what is the purpose of this `description` attribute in the `div` tag? Is it used somewhere? 
Otherwise, I would prefer not to add additional attributes (in fact, I don't see this is a global [html attribute](https://www.w3schools.com/tags/ref_standardattributes.asp))

Reply:
Got it, thanks for the clarification. Nothing to change then ;-)